### ISSUE TITLE
Where is the script "layupDesign_ANSYSmesh.m", "layupDesign_ANSYSanalysis.m"

### ISSUE THREAD

**Issue by namnguyen-lqdtu** [sandialabs/NuMAD#37]
In the user's guide at this link: https://numad.readthedocs.io/en/latest/user/fea.html, some scripts are mentioned (for examples: "layupDesign_ANSYSmesh.m", "layupDesign_ANSYSanalysis.m") but they dont exist in the source code. Please provide those codes with example of using them.

**Comment by namnguyen-lqdtu** [comment]
Thank you @ecamare! 
I have downloaded the NuMAD-dev code and found the scripts. I am now  trying to run the scripts in the "example" folder. For the example in the "runIEC" folder, when running the scripts "IECLoadsAnalysis.m", the line 33 doesn't work. It goes to Debugging mode and stops at the line 68 in the function "resampleAirfoilDB", waiting for an input from keyboard (see pictures below). What do I have to do now?
![image](https://github.com/sandialabs/NuMAD/assets/84730230/cc825a1f-521c-4dc5-9ca7-66eacdf95bbe)
![image](https://github.com/sandialabs/NuMAD/assets/84730230/bf4a553f-6c1b-4270-beb7-26080d60206b)

**Comment by kbonney** [comment]
@namnguyen-lqdtu Seems like you are encountering a remnant of some debugging code. This is something we should address, but in the mean time I believe you can get around this by just pressing the continue button located in the top toolbar.

**Comment by moonstar123456** [comment]
> Thank you @ecamare! I have downloaded the NuMAD-dev code and found the scripts. I am now trying to run the scripts in the "example" folder. For the example in the "runIEC" folder, when running the scripts "IECLoadsAnalysis.m", the line 33 doesn't work. It goes to Debugging mode and stops at the line 68 in the function "resampleAirfoilDB", waiting for an input from keyboard (see pictures below). What do I have to do now? ![image](https://user-images.githubusercontent.com/84730230/241243915-cc825a1f-521c-4dc5-9ca7-66eacdf95bbe.png) ![image](https://user-images.githubusercontent.com/84730230/241244095-bf4a553f-6c1b-4270-beb7-26080d60206b.png)

Hello, Nice to meet you! i am learning numad 3.0 and have some questions, is it possible to consult you? My email is 843526493@qq.com, skype: emilymoonjiang.   Many thanks in advance!

**Comment by namnguyen-lqdtu** [comment]
Yes. You can come back to me in October. I am too busy in September.Sent from my iPhoneOn 18 Sep 2023, at 09:31, moonstar123456 ***@***.***> wrote:﻿

Thank you @ecamare! I have downloaded the NuMAD-dev code and found the scripts. I am now trying to run the scripts in the "example" folder. For the example in the "runIEC" folder, when running the scripts "IECLoadsAnalysis.m", the line 33 doesn't work. It goes to Debugging mode and stops at the line 68 in the function "resampleAirfoilDB", waiting for an input from keyboard (see pictures below). What do I have to do now?  

Hello, Nice to meet you! i am learning numad 3.0 and have some questions, is it possible to consult you? My email is ***@***.***, skype: emilymoonjiang.   Many thanks in advance!
